src/metabase/db/custom_migrations.clj

```diff
@@ -277,21 +288,38 @@
                                  _ [[k v]]))
                              column_settings)))))))
 
-(define-migration AddJoinAliasToVisualizationSettingsFieldRefs
+(define-reversible-migration AddJoinAliasToColumnSettingsFieldRefs
+  (let [update-one! (fn [{:keys [id visualization_settings] :as card}]
```

Comment:
indent the body of the fn and/or use a `letfn`

Reply:
Fixed [Fix indentation and naming consistency](https://github.com/metabase/metabase/pull/31150/commits/f4ed0cc5967bb13a94d51e88ccfc7c31efcd9f59)